src/io.c

```diff
@@ -685,37 +685,39 @@ send_buffered_event (gpointer event, gpointer data)
 gboolean
 line_buffer_io (GIOChannel *gio, GIOCondition condition, gpointer data)
 {
-    UZBL_UNUSED (condition);
-
     UzblIOBufferData *io_data = (UzblIOBufferData *)data;
 
     static const gsize BUFSZ = 256;
     gchar input[BUFSZ];
     gsize read;
     GError *error = NULL;
-    gboolean eof = FALSE;
 
-    do {
-        GIOStatus status = g_io_channel_read_chars (gio, input, BUFSZ, &read, &error);
+    if (condition > 10) {
```

Comment:
ah yes. just debug stuff from trying to figure out what the conditions I got back was. Should be removed